Library/Homebrew/shims/linux/super/cc

```diff
@@ -359,6 +359,7 @@ if __FILE__ == $PROGRAM_NAME
     path = Pathname.new(x)
     path.directory? && path.realpath == superbin
   end
+  paths << "#{ENV["HOMEBREW_PREFIX"]}/bin"
```

Comment:
`@prefix` is a variable inside cmd class. But we are outside cmd class here. Also it seems that `ENV["HOMEBREW_PREFIX"]` is also used other place in this script.